### DISCUSSION TITLE
Glowroot for other tech stacks

### DISCUSSION THREAD

**Discussion by Shinjiru-stack** [glowroot/glowroot#917] [General]
Does glowroot support another tech stack?
If not, is there any plan for that?

**Comment by Never-lab** [comment]
Glowroot is a **JVM bytecode agent** — it monitors Java (and other JVM languages: Kotlin, Scala, Groovy, etc.) by weaving into the running process. There is no support for non-JVM stacks (.NET, Node, Python, Go, …) and no public roadmap for that: the core value is Java-specific weaving plus plugins (Servlet, Spring, JDBC, …).

If you meant “does it work with framework X on the JVM?”, check the plugins under `agent/plugins/` or ask with the exact stack — many Java frameworks are already covered.

(FYI this kind of question fits **Q&A** best — see the channel map in #1223.)
